runtime/criusupport/criusupport.cpp

```diff
@@ -279,8 +279,11 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 		char workDirBuf[STRING_BUFFER_SIZE];
 		char *workDirChars = workDirBuf;
 		BOOLEAN isAfterCheckpoint = FALSE;
-		I_64 beforeCheckpoint = 0;
-		I_64 afterRestore = 0;
+		I_64 checkpointNanoTimeMononic = 0;
+		I_64 restoreNanoTimeMononic = 0;
+		U_64 checkpointNanoRealTime = 0;
```

Comment:
perhaps rename to UTC time

Reply:
Renamed to `checkpointNanoUTCTime`/`restoreNanoUTCTime`.